package.json

```diff
@@ -30,7 +32,9 @@
     "expo-sharing": "~10.0.3",
     "expo-status-bar": "~1.1.0",
     "expo-updates": "~0.10.15",
+    "graphql": "^14.0.0",
     "lodash": "^4.17.21",
+    "prop-types": "^15.8.1",
```

Comment:
prop-types is useless with typescript. Typescript better improves types than prop-types